chapters/hpmor-chapter-035.tex

```diff
@@ -125,7 +125,9 @@
 
 And a magical civilization, slowly fading as knowledge was lost, still governed by a nobility that saw Muggles as not quite human.
 
-It was a terribly sad feeling, but not one that held any hint of doubt. \latersection{Aftermath: Blaise Zabini.}
+It was a terribly sad feeling, but not one that held any hint of doubt.
+
+\latersection{Aftermath: Blaise Zabini.}
```

Comment:
I think in these `\latersection`s the best final punctuation is nothing (i.e. here, not a full stop).

Reply:
I agree, think this is now fixed in another PR, but now fixed here as well ;-)